featuretools/primitives/standard/transform/numeric/rate_of_change.py

```diff
@@ -0,0 +1,35 @@
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Double
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class RateOfChange(TransformPrimitive):
+    """Computes the rate of change of a value per second.
```

Comment:
I think it may be interesting in the future to have the unit of time be configurable